### ISSUE TITLE
__round__ support

### ISSUE THREAD

**Issue by brandtbucher** [InvestmentSystems/static-frame#198]
SF should support `__round__` everywhere it already supports `__abs__`.

It should be pretty straightforward to just delegate to `numpy.round` by adding an entry [here](https://github.com/InvestmentSystems/static-frame/blob/master/static_frame/core/container.py#L341-L358), right?

**Comment by flexatone** [comment]
Interestingly, NP does not support `__round__`, even though it supports `__abs__`. So this will have to be implemented more directly than using those operator deriven function generators.

```
>>> round(sf.Series([3,4,5.0001]).values, 2)
Traceback (most recent call last):
  File "<console>", line 1, in <module>
TypeError: type numpy.ndarray doesn't define __round__ method
```

**Comment by flexatone** [comment]
And while defining this in `UFUNC_SHAPE_SKIPNA` would almost work, that family of functions all expect an `axis` argument when applied to a 2D array; such an argument does not really make sense for `np.round`.

This case is a bit more like `astype()` methods; on a `Frame`, we might want to expose column selection to only round a subset of columns.

**Comment by flexatone** [comment]
Support for `round` is completed on `Series` and will be part of the 0.6.11 release. Support for `round` on `Frame` will be part of 0.6.12.

**Comment by flexatone** [comment]
I have completed adding support for `__round__` to `Frame`. For now, I only exposed this via the built-in `round` function, rather than exposing a more powerful interface that permits sub-set column selection. This will obviously only work for numeric types.

This will be made available as part of the 0.6.15 release (a bit later than expected).

```python
>>> f = sf.Frame(np.full((3,4), .33333))
>>> f
<Frame>
<Index> 0         1         2         3         <int64>
<Index>
0       0.33333   0.33333   0.33333   0.33333
1       0.33333   0.33333   0.33333   0.33333
2       0.33333   0.33333   0.33333   0.33333
<int64> <float64> <float64> <float64> <float64>

>>> round(f, 2)
<Frame>
<Index> 0         1         2         3         <int64>
<Index>
0       0.33      0.33      0.33      0.33
1       0.33      0.33      0.33      0.33
2       0.33      0.33      0.33      0.33
<int64> <float64> <float64> <float64> <float64>
```
